core/services/keystore/eth.go

```diff
@@ -177,19 +177,45 @@ func (ks *eth) Export(id string, password string) ([]byte, error) {
 }
 
 // Get the next nonce for the given key and chain. It is safest to always to go the DB for this
-func (ks *eth) GetNextNonce(address common.Address, chainID *big.Int, qopts ...pg.QOpt) (int64, error) {
+func (ks *eth) GetNextNonce(address common.Address, chainID *big.Int, qopts ...pg.QOpt) (nonce int64, err error) {
 	if !ks.exists(address) {
 		return 0, errors.Errorf("key with address %s does not exist", address.Hex())
 	}
-	return ks.orm.getNextNonce(address, chainID, qopts...)
+	nonce, err = ks.orm.getNextNonce(address, chainID, qopts...)
```

Comment:
You don't check `err`, and do not use it anywhere below... looks odd.

Reply:
Fixed, thanks